index.html

```diff
@@ -9,19 +9,35 @@
   <title> Portfolio tite</title>
   <link rel="stylesheet" href="styles.css">
   <script src="https://kit.fontawesome.com/213f6776ae.js" crossorigin="anonymous"></script>
+  <script src="script.js" defer></script>
 </head>
 
 <body>
   <header class="navtools_container">
+   
     <h2 class="logo">Tite</h2>
+    <ul class="mobile-menu">
+      <li class="mobile-items">
+        <a href="#home" class="mobile-link">Portfolio</a>
+      </li>
+      <li class="mobile-items">
+        <a href="#about" class="mobile-link">About</a>
+      </li>
+      <li class="mobile-items">
+        <a href="#contact" class="mobile-link">Contact</a>
+      </li>
+    </ul>
+
+      <div class="hamburger navbar">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+      </div>
+
 
-    <nav class="navbar">
-    <nav class="navbar">
-      <i class="fa-solid fa-bars"></i>
-    </nav>
   </header>
 
-  <main class="main_container">
+  <main class="main_container" id="home">

```

Comment:
- [ ] Kindly remove this `id` attribute from here and add the `id` attribute in `line no. 73`  then if someone click on the `Portfolio` link it will scroll down to the `Portfolio` or ` My Recent work` section :+1: 

```suggestion
  <main class="main_container">

```